src/labels/CustomLabels.labels

```diff
@@ -8432,6 +8432,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Video Duration displayed in the Video Header Component</shortDescription>
         <value>in 2 minutes</value>
     </labels>
+    <labels>
```

Comment:
Can you place these in the same labels file alphabetically? There are other labels that start with `gseu` so these new labels should fit in with them.